virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -14,39 +14,26 @@ The Scheduling, Scale, and Performance (SSP) Operator integrates {VirtProductNam
 * Run commands in VMs
 * Manipulate disk images with `libguestfs` tools
 
-:FeatureName: Managing virtual machines with {pipelines-title}
-include::snippets/technology-preview.adoc[]
-
-
 [id="prerequisites_virt-managing-vms-openshift-pipelines"]
 == Prerequisites
 
 * You have access to an {product-title} cluster with `cluster-admin` permissions.
 * You have installed the OpenShift CLI (`oc`).
 * You have link:https://docs.openshift.com/pipelines/latest/install_config/installing-pipelines.html[installed {pipelines-shortname}].
 
-
-include::modules/virt-deploying-ssp.adoc[leveloffset=+1]
-
 include::modules/virt-supported-ssp-tasks.adoc[leveloffset=+1]
 
-
 [id="example-pipelines_virt-managing-vms-openshift-pipelines"]
-== Example pipelines
-
-The SSP Operator includes the following example `Pipeline` manifests. You can run the example pipelines by using the web console or CLI.
-
-You might have to run more than one installer pipeline if you need multiple versions of Windows. If you run more than one installer pipeline, each one requires unique parameters, such as the `autounattend` config map and base image name. For example, if you need Windows 10 and Windows 11 or Windows Server 2022 images, you have to run both the Windows efi installer pipeline and the Windows bios installer pipeline. However, if you need Windows 11 and Windows Server 2022 images, you have to run only the Windows efi installer pipeline.
-
-Windows EFI installer pipeline:: This pipeline installs Windows 11 or Windows Server 2022 into a new data volume from a Windows installation image (ISO file). A custom answer file is used to run the installation process.
+[id="Windows-EFI-installer-pipeline-virt-managing-vms-openshift-pipelines"]
+== Windows EFI installer pipeline
```

Comment:
Can you please add mention, that the pipeline can be found here: https://artifacthub.io/packages/tekton-pipeline/redhat-pipelines/windows-efi-installer